resources/ctl.rb

```diff
@@ -73,27 +72,11 @@
         not_if "Test-Path c:/sensutemp/sensu-go_#{node['sensu-go']['ctl_version']}_windows_amd64.tar.gz"
       end
 
-      seven_zip_archive 'Extract Sensuctl Gz' do
-        path "c:/sensutemp/sensu-go_#{node['sensu-go']['ctl_version']}_windows_amd64.tar"
-        source "c:/sensutemp/sensu-go_#{node['sensu-go']['ctl_version']}_windows_amd64.tar.gz"
-        overwrite true
-        timeout   30
-      end
-
-      seven_zip_archive 'Extract Sensuctl Tar' do
-        path "c:/sensutemp/sensu-go_#{node['sensu-go']['ctl_version']}_windows_amd64"
-        source "c:/sensutemp/sensu-go_#{node['sensu-go']['ctl_version']}_windows_amd64.tar"
-        overwrite true
-        timeout   30
-      end
-
-      directory sensuctl_bin do
-        recursive true
-      end
-
-      remote_file "#{sensuctl_bin}\\sensuctl.exe" do
-        source "file:///c:/sensutemp/sensu-go_#{node['sensu-go']['ctl_version']}_windows_amd64/sensuctl.exe"
-      end
+    archive_file 'Extract Sensuctl' do
```

Comment:
I am thinking we should have an action of nothing and have the powershell script that downloads the script notify this instead, it already has a logical guard to not download again if the same version already exists.

Reply:
refactored to use notify